Protect SDM disconnectWriter

cmft opened the pull request:
SMD.disconnectWriter fails raising an Exception if a signal is not
previously connected. It happens since taurus uses the new API
signal.

A protection is needed to disconnectWriter. Add it.

reszelaz commented:
Some comments:

- Link in the designer guide does not work: taurus.qt.qtgui.util.ui
- Difficulties in using scripts in the develop mode is not explained neither warned.
- Taurus web was removed but it is used in the "Users introduction"
- One example in "Users introduction" uses python taurusform.py instead of taurusform. Is it on purpose?
- In "Forms" Sardana examples on custom widgets could be better explained
- "Color meaning" is aligned to the center.
- Qualities warning and alarm are squashed to alarm only.

cpascual commented:
>  - Link in the designer guide does not work: taurus.qt.qtgui.util.ui

Fixed

> - Difficulties in using scripts in the develop mode is not explained neither warned.

There are no difficulties now. It works well.

> - Taurus web was removed but it is used in the "Users introduction"

Fixed

> - One example in "Users introduction" uses python taurusform.py instead of taurusform. Is it on purpose?

Fixed

> - In "Forms" Sardana examples on custom widgets could be better explained

Done

> - "Color meaning" is aligned to the center.

Fixed

> - Qualities warning and alarm are squashed to alarm only.

Fixed

cpascual commented:
For my part, I consider the changes finished. Please review last changes and, if ok, proceed to merge.

reszelaz commented:
I still see problems with alignment of "Color meaning". I see it aligned to the center (both in my local build and in: http://www.taurus-scada.org/en/bug-221/users/ui/ui_colors.html)

cpascual commented:
> I still see problems with alignment of "Color meaning".

Strange. Firefox renders it ok but chrome does not. It is likely related to some wrong html from the tables in this page. I suggest to treat it as a bug for the future

cmft committed:
Protect SDM disconnectWriter

SMD.disconnectWriter fails raising an Exception if a signal is not
previously connected. It happens since taurus uses the new API
signal.

A protection is needed to disconnectWriter. Add it.
